app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -76,6 +76,8 @@ public class DocumentEditAndViewFragment extends MarkorBaseFragment implements F
     public static final String SAVESTATE_DOCUMENT = "DOCUMENT";
     public static final String START_PREVIEW = "START_PREVIEW";
 
+    public static float VIEW_FONT_SCALE = 100f / 15.7f;
```

Comment:
Thanks!

Do you see that scale still fitting quite good?

Reply:
Looks good to me. But I do all my testing just my own primary device - a pixel 7 pro